pkg/kev/config/schema.go

```diff
@@ -27,14 +27,15 @@ var ServicesSchema = map[string]interface{}{
 		LabelWorkloadImagePullPolicy:                     map[string]interface{}{"type": "string", "pattern": `^(?i)(IfNotPresent|Always)$`},
 		LabelWorkloadImagePullSecret:                     map[string]interface{}{"type": "string"},
 		LabelWorkloadRestartPolicy:                       map[string]interface{}{"type": "string", "pattern": `^(?i)(Always|OnFailure|Never)$`},
+		LabelWorkloadLivenessProbeType:                   map[string]interface{}{"type": "string", "enum": []string{"none", "command", "tcp", "http"}},
```

Comment:
Should we remove "tcp" in this PR as it's not implemented yet?

Reply:
yep 👍

Reply:
@mangas this is still pending 😉